pkg/debug/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV BUILD_PKGS abuild curl tar make linux-headers patch g++ git gcc ncurses-dev
 # forget to check on all supported architectures: e.g. arm64
```

Comment:
@shjala you left two Signed-off-by in your commit's message. Also `BusyBox` is written without a space.

Reply:
fixed.

Reply:
it seems `Busy Box` with space is still there

Reply:
> Also BusyBox is written without a space.

you said it should be with space, so I added.

Reply:
I guess in the commit message...
Just in case, I observed the strange behavior of GitHub yesterday. I had pushed some changes, and GitHub showed them to me fine, but the other reviewers didn't see them. And after the merge - they were not taken into account.

Reply:
Ah, sorry! I guess the original comment from @rene was that the right spelling of BusyBox is without a space =) I guess it was a misunderstanding))

Reply:
changed it to "BusyBox" again.